code/modules/projectiles/projectile/beams.dm

```diff
@@ -8,6 +8,9 @@
 	flag = "laser"
 	eyeblur = 2
 
+/obj/item/projectile/beam/hosbeam
+	damage = 25

```

Comment:
Mmm, looks like we've got another coder who tells lies about their PR. "20 damage" my ass.